lib/vm/src/export.rs

```diff
@@ -9,6 +9,22 @@ use crate::vmcontext::{VMFunctionBody, VMFunctionEnvironment, VMFunctionKind, VM
 use std::sync::Arc;
 use wasmer_types::{FunctionType, MemoryType, TableType};
 
+/// The value of an export passed from one instance to another.
+#[derive(Debug, Clone)]
+pub enum EngineExport {
```

Comment:
This should live in the engine

Reply:
Any thoughts on the fact that `EngineExport` seems to entirely replace the use of `Export`?